src/minecraft/kotlin/xyz/wagyourtail/unimined/internal/minecraft/patch/reindev/NoTransformReIndevTransformer.kt

```diff
@@ -0,0 +1,25 @@
+package xyz.wagyourtail.unimined.internal.minecraft.patch.reindev
+
+import org.gradle.api.Project
+import xyz.wagyourtail.unimined.api.mapping.MappingNamespaceTree
+import xyz.wagyourtail.unimined.api.minecraft.EnvType
+import xyz.wagyourtail.unimined.api.minecraft.MinecraftJar
+import xyz.wagyourtail.unimined.api.unimined
+import xyz.wagyourtail.unimined.internal.minecraft.MinecraftProvider
+import xyz.wagyourtail.unimined.internal.minecraft.patch.AbstractMinecraftTransformer
+import xyz.wagyourtail.unimined.util.*
+import java.net.URI
+import java.nio.file.Path
+import kotlin.io.path.createDirectories
+import kotlin.io.path.exists
+import kotlin.time.Duration
+
+class NoTransformReIndevTransformer(project: Project, provider: ReIndevProvider): AbstractReIndevTransformer(
+    project,
+    provider,
+    "ReIndev-none"
+) {
+
+    override var prodNamespace: MappingNamespaceTree.Namespace = provider.mappings.OFFICIAL
```

Comment:
This actually does nothing, the same way it does nothing in `NoTransformMinecraftTransformer` cc @wagyourtail